packages/buckets/package.json

```diff
@@ -30,9 +30,9 @@
   "license": "MIT",
   "dependencies": {
     "@improbable-eng/grpc-web": "^0.13.0",
-    "@textile/buckets-grpc": "2.1.7",
     "@textile/context": "^0.9.2",
     "@textile/crypto": "^1.0.1",
+    "@textile/buckets-grpc": "2.1.4",
```

Comment:
Downgrade?

Reply:
not sure what happened there. npm link hiccup i think